src/mistral-client.ts

```diff
@@ -1,132 +1,121 @@
-const VERSION = '0.0.3';
```

Comment:
Ideally this PR would be based on top of the latest commit in main. It doesn't seem to be, given VERSION of 0.0.3 ?

Reply:
but it is set `0.2.0` in latest commit in main branch:
https://github.com/mistralai/client-js/blob/2fcc56418b11293b23c244b02dd1fce70695405d/src/client.js#L1

or anything I misunderstand?

Reply:
Right, when looking at the diff for this PR we are seeing 0.0.3 
That's because this branch is commits based on an old version of main
It complicates review a little bit